app/src/main/java/com/aerogear/androidshowcase/MainActivity.java

```diff
@@ -107,7 +108,11 @@ protected void onCreate(Bundle savedInstanceState) {
 
         testNetwork();
         // load the main menu fragment
-        navigator.navigateToHomeView(this, getString(R.string.fragment_title_home));
+        if(getIntent().getStringExtra(UnifiedPushMessage.class.getName()) == null) {
```

Comment:
👍

Reply:
`onActivityResult()` does not work because who is calling this Activity is a [service running in the background](https://github.com/aerogear/android-showcase-template/blob/master/app/src/main/java/com/aerogear/androidshowcase/handler/NotificationBarMessageHandler.java#L69-L70) not a return from an Activity.